UDPServer/SiceUDPServer.cs

```diff
@@ -85,5 +84,47 @@ public void Dispose()
         {
             client.Dispose();
         }
+
+        public void Handle(ServerCommand command)
+        {
+            if(command.Command)
+            {
+                ConnectionIP = command.ConnectionIP;
+                ConnectionPort = command.ConnectionPort;
+                this.Connect();
+            }
+            else
+            {
+                this.Disconnect();
+            }
+        }
+
+        public class ServerCommand
+        {
+            public ServerCommand(bool command, string IP, string port)
+            {
+                this.Command = command;
+                if(IPAddress.TryParse(IP, out IPAddress parseIP))
+                {
+                    ConnectionIP = parseIP;
+                }
+                if(int.TryParse(port, out int parsePort))
+                {
+                    ConnectionPort = parsePort;
+                }
+            }
+            public bool Command { get; set; }
+            public IPAddress ConnectionIP { get; set; }
+            public int ConnectionPort { get; set; }
+        }
```

Comment:
Keep classes in separate files for best practice.

Reply:
Changed